### PR TITLE
Fixed showing -2 damage when hydras attack shadow assasins and they e…

### PR DESCRIPTION
**Pull request by Kert**
…vade the attack

### CODE REVIEW

File: src/cpp/shared/combat/army.cpp [hunk 1]
```diff
@@ -589,6 +597,69 @@ void army::DoAttack(int isRetaliation) {
 
 }
 
+void army::DoHydraAttack(int isRetaliation) {
```
Comment (jkoppel) [inline src/cpp/shared/combat/army.cpp]: Can you add a comment about what you changed from the original code?

Also, please explain the bug.

Comment (Kert) [reply src/cpp/shared/combat/army.cpp]: added a comment
It was just a problem with displaying combat message

**Comment by jkoppel** [comment]
How can a hydra do < 0 damage?

**Comment by Kert** [comment]
> How can a hydra do < 0 damage?

Attack a mirror image and destroy it (-1 damage) or attack assasin and he evades (-2 damage)

**Comment by jkoppel** [comment]
Why are those negative damage? Where is this set?

**Comment by Kert** [comment]
> Why are those negative damage? Where is this set?

army::DamageEnemy

**Pull request merged**